client/header/activity-panel/panels/inbox/style.scss

```diff
@@ -1,9 +1,8 @@
-
-
 .woocommerce-inbox-message {
 	position: relative;
 	color: $gray-text;
-	background: $core-grey-light-200;
+	// background: $gray-100;
```

Comment:
commented out code

Reply:
This looks a bit weird. some mistake in merging the other PR to this, I'm not sure, either way thanks for spotting, will fix.